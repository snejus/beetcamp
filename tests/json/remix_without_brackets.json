{
  "@context": "https://schema.org",
  "@id": "https://purehate000.bandcamp.com/album/aluphobia-x-astatine-problem-child-ep",
  "@type": "MusicAlbum",
  "additionalProperty": [
    {
      "@type": "PropertyValue",
      "name": "art_id",
      "value": 237465596
    },
    {
      "@type": "PropertyValue",
      "name": "featured_track_num",
      "value": 3
    },
    {
      "@type": "PropertyValue",
      "name": "license_name",
      "value": "all_rights_reserved"
    }
  ],
  "albumRelease": [
    {
      "@id": "https://purehate000.bandcamp.com/album/aluphobia-x-astatine-problem-child-ep",
      "@type": [
        "MusicRelease",
        "Product"
      ],
      "additionalProperty": [
        {
          "@type": "PropertyValue",
          "name": "item_id",
          "value": 1784256378
        },
        {
          "@type": "PropertyValue",
          "name": "item_type",
          "value": "a"
        },
        {
          "@type": "PropertyValue",
          "name": "selling_band_id",
          "value": 241766404
        },
        {
          "@type": "PropertyValue",
          "name": "type_name",
          "value": "Digital"
        },
        {
          "@type": "PropertyValue",
          "name": "art_id",
          "value": 237465596
        }
      ],
      "description": "Includes high-quality download in MP3, FLAC and more. Paying supporters also get unlimited streaming via the free Bandcamp app.",
      "image": [
        "https://f4.bcbits.com/img/a0237465596_10.jpg"
      ],
      "musicReleaseFormat": "DigitalFormat",
      "name": "Aluphobia x Astatine - Problem Child EP"
    },
    {
      "@id": "https://purehate000.bandcamp.com/album/aluphobia-x-astatine-problem-child-ep#b57237545",
      "@type": [
        "MusicRelease",
        "Product"
      ],
      "additionalProperty": [
        {
          "@type": "PropertyValue",
          "name": "item_id",
          "value": 57237545
        },
        {
          "@type": "PropertyValue",
          "name": "item_type",
          "value": "b"
        },
        {
          "@type": "PropertyValue",
          "name": "selling_band_id",
          "value": 241766404
        },
        {
          "@type": "PropertyValue",
          "name": "type_name",
          "value": "Digital"
        },
        {
          "@type": "PropertyValue",
          "name": "art_id",
          "value": 3901033007
        },
        {
          "@type": "PropertyValue",
          "name": "is_bfd",
          "value": true
        }
      ],
      "image": [
        "https://f4.bcbits.com/img/a3901033007_10.jpg"
      ],
      "musicReleaseFormat": "DigitalFormat",
      "name": "full digital discography (63 releases)",
      "offers": {
        "@type": "Offer",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "bundle_size",
            "value": 63
          },
          {
            "@type": "PropertyValue",
            "name": "discount",
            "value": 0.7
          }
        ],
        "availability": "OnlineOnly",
        "price": 74.4,
        "priceCurrency": "EUR",
        "priceSpecification": {
          "minPrice": 74.4
        },
        "url": "https://purehate000.bandcamp.com/album/aluphobia-x-astatine-problem-child-ep#b57237545-buy"
      }
    },
    {
      "@id": "https://purehate000.bandcamp.com/track/aluphobia-x-astatine-problem-child",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://purehate000.bandcamp.com/track/aluphobia-x-astatine-problem-child-illiya-korniyen-remix",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://purehate000.bandcamp.com/track/aluphobia-x-astatine-problem-child-korcs-remix",
      "@type": "MusicRelease"
    }
  ],
  "albumReleaseType": "AlbumRelease",
  "byArtist": {
    "@id": "https://purehate000.bandcamp.com",
    "@type": "MusicGroup",
    "name": "PURE HATE"
  },
  "copyrightNotice": "All Rights Reserved",
  "dateModified": "10 Dec 2021 15:36:56 GMT",
  "datePublished": "10 Dec 2021 00:00:00 GMT",
  "image": "https://f4.bcbits.com/img/a0237465596_10.jpg",
  "keywords": [
    "Electronic",
    "HardTechno",
    "industrialtechno",
    "techno",
    "Budapest"
  ],
  "mainEntityOfPage": "https://purehate000.bandcamp.com/album/aluphobia-x-astatine-problem-child-ep",
  "name": "Aluphobia x Astatine - Problem Child EP",
  "numTracks": 3,
  "publisher": {
    "@id": "https://purehate000.bandcamp.com",
    "@type": "MusicGroup",
    "additionalProperty": [
      {
        "@type": "PropertyValue",
        "name": "band_id",
        "value": 241766404
      },
      {
        "@type": "PropertyValue",
        "name": "has_any_downloads",
        "value": true
      },
      {
        "@type": "PropertyValue",
        "name": "image_height",
        "value": 2126
      },
      {
        "@type": "PropertyValue",
        "name": "image_id",
        "value": 20591482
      },
      {
        "@type": "PropertyValue",
        "name": "image_width",
        "value": 2126
      }
    ],
    "description": "Hungary based event series / movement / independent label",
    "foundingLocation": {
      "@type": "Place",
      "name": "Budapest, Hungary"
    },
    "genre": "https://bandcamp.com/discover/electronic",
    "image": "https://f4.bcbits.com/img/0020591482_10.jpg",
    "mainEntityOfPage": [
      {
        "@type": "WebPage",
        "name": "SoundCloud",
        "url": "https://soundcloud.com/purehate000"
      }
    ],
    "name": "PURE HATE",
    "subjectOf": [
      {
        "@type": "WebPage",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "nav_type",
            "value": "m"
          }
        ],
        "name": "NOISE",
        "url": "https://purehate000.bandcamp.com/music"
      },
      {
        "@type": "WebPage",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "nav_type",
            "value": "p"
          }
        ],
        "name": "CLOTHING/12\"",
        "url": "https://purehate000.bandcamp.com/merch"
      },
      {
        "@type": "WebPage",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "nav_type",
            "value": "c"
          }
        ],
        "name": "COMMUNITY",
        "url": "https://purehate000.bandcamp.com/community"
      }
    ]
  },
  "track": {
    "@type": "ItemList",
    "itemListElement": [
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://purehate000.bandcamp.com/track/aluphobia-x-astatine-problem-child",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 441026690
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H05M12S",
          "mainEntityOfPage": "https://purehate000.bandcamp.com/track/aluphobia-x-astatine-problem-child",
          "name": "Aluphobia x Astatine - Problem Child"
        },
        "position": 1
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://purehate000.bandcamp.com/track/aluphobia-x-astatine-problem-child-illiya-korniyen-remix",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 897535301
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H06M35S",
          "mainEntityOfPage": "https://purehate000.bandcamp.com/track/aluphobia-x-astatine-problem-child-illiya-korniyen-remix",
          "name": "Aluphobia x Astatine - Problem Child Illiya Korniyen Remix"
        },
        "position": 2
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://purehate000.bandcamp.com/track/aluphobia-x-astatine-problem-child-korcs-remix",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 2655269613
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H06M31S",
          "mainEntityOfPage": "https://purehate000.bandcamp.com/track/aluphobia-x-astatine-problem-child-korcs-remix",
          "name": "Aluphobia x Astatine - Problem Child Korcs Remix"
        },
        "position": 3
      }
    ],
    "numberOfItems": 3
  }
}
