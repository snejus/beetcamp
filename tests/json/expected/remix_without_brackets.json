[
  {
    "album": "Problem Child EP",
    "album_id": "https://purehate000.bandcamp.com/album/aluphobia-x-astatine-problem-child-ep",
    "artist": "Aluphobia, Astatine",
    "artist_id": "https://purehate000.bandcamp.com",
    "tracks": [
      {
        "title": "Problem Child",
        "track_id": "https://purehate000.bandcamp.com/track/aluphobia-x-astatine-problem-child",
        "release_track_id": null,
        "artist": "Aluphobia x Astatine",
        "artist_id": "https://purehate000.bandcamp.com",
        "length": 312,
        "index": 1,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 1,
        "medium_total": 3,
        "artist_sort": null,
        "disctitle": null,
        "artist_credit": null,
        "data_source": "bandcamp",
        "data_url": "https://purehate000.bandcamp.com/album/aluphobia-x-astatine-problem-child-ep",
        "lyricist": null,
        "composer": null,
        "composer_sort": null,
        "arranger": null,
        "track_alt": null,
        "work": null,
        "mb_workid": null,
        "work_disambig": null,
        "bpm": null,
        "initial_key": null,
        "genre": null
      },
      {
        "title": "Problem Child (Illiya Korniyen Remix)",
        "track_id": "https://purehate000.bandcamp.com/track/aluphobia-x-astatine-problem-child-illiya-korniyen-remix",
        "release_track_id": null,
        "artist": "Aluphobia x Astatine",
        "artist_id": "https://purehate000.bandcamp.com",
        "length": 395,
        "index": 2,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 2,
        "medium_total": 3,
        "artist_sort": null,
        "disctitle": null,
        "artist_credit": null,
        "data_source": "bandcamp",
        "data_url": "https://purehate000.bandcamp.com/album/aluphobia-x-astatine-problem-child-ep",
        "lyricist": null,
        "composer": null,
        "composer_sort": null,
        "arranger": null,
        "track_alt": null,
        "work": null,
        "mb_workid": null,
        "work_disambig": null,
        "bpm": null,
        "initial_key": null,
        "genre": null
      },
      {
        "title": "Problem Child (Korcs Remix)",
        "track_id": "https://purehate000.bandcamp.com/track/aluphobia-x-astatine-problem-child-korcs-remix",
        "release_track_id": null,
        "artist": "Aluphobia x Astatine",
        "artist_id": "https://purehate000.bandcamp.com",
        "length": 391,
        "index": 3,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 3,
        "medium_total": 3,
        "artist_sort": null,
        "disctitle": null,
        "artist_credit": null,
        "data_source": "bandcamp",
        "data_url": "https://purehate000.bandcamp.com/album/aluphobia-x-astatine-problem-child-ep",
        "lyricist": null,
        "composer": null,
        "composer_sort": null,
        "arranger": null,
        "track_alt": null,
        "work": null,
        "mb_workid": null,
        "work_disambig": null,
        "bpm": null,
        "initial_key": null,
        "genre": null
      }
    ],
    "asin": null,
    "albumtype": "ep",
    "va": false,
    "year": 2021,
    "month": 12,
    "day": 10,
    "label": "PURE HATE",
    "mediums": 1,
    "artist_sort": null,
    "releasegroup_id": null,
    "catalognum": "",
    "script": null,
    "language": null,
    "country": "HU",
    "style": "electronic",
    "genre": "techno",
    "albumstatus": "Official",
    "media": "Digital Media",
    "albumdisambig": null,
    "releasegroupdisambig": null,
    "artist_credit": null,
    "original_year": null,
    "original_month": null,
    "original_day": null,
    "data_source": "bandcamp",
    "data_url": "https://purehate000.bandcamp.com/album/aluphobia-x-astatine-problem-child-ep",
    "discogs_albumid": null,
    "discogs_labelid": null,
    "discogs_artistid": null,
    "comments": null,
    "albumtypes": "ep; single"
  }
]
